lm_eval/api/task.py

```diff
@@ -1098,13 +1101,20 @@ def process_results(self, doc, results):
                 acc_norm = 1.0 if pred_norm == gold else 0.0
                 # TODO: this gets score of 0 on arc_challenge for pythia-70m. need to test that this works properly
                 exact_match = int(is_greedy[gold]) if gold != -100 else 0
+                if gold == -100:
+                    brier = 0
+                else:
+                    labels = np.zeros_like(pred_probs)
+                    labels[gold] = 1
+                    brier = np.mean(np.square(pred_probs - labels))
```

Comment:
nit: I think this is technically the normalized brier score since we're normalizing by the number of choices.

Reply:
yeah, I think that distinction is fine though as long as we know that it's the version we're using